### PR TITLE
Use organization webhook to log Project v2 events

### LINKED ISSUE #88: Use organization webhooks and log them
We want our [Conform PR GitHub Action](https://github.com/FerretDB/github-actions/tree/main/conform-pr) to check PR's Project fields after they are updated, but there is no such [event to trigger an Action run](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows). That's because Projects v2 are organization-scoped, and Actions are repository-scoped.

The solution is to install an organization webhook to receive Project v2 events, find matching PR, and schedule a new Conform PR run.

Let's create a simple service for it. It should use packages we already use (github.com/google/go-github and github.com/shurcooL/githubv4); it should not use any web or API framework. It should receive incoming webhooks, validate them by [checking the signature](https://docs.github.com/en/developers/webhooks-and-events/webhooks/securing-your-webhooks#validating-payloads-from-github), check secrets, ~find matching PR and schedule a new Conform PR run~ and just log them for now.

Webhook example:

```
X-Github-Event: projects_v2_item
X-Hub-Signature: sha1=5d9d657f2244700cab8e7996af2ce3ef476d5016
X-Hub-Signature-256: sha256=9c23e56fdae920dc6eb3754cd4f4272e1909b24da6d4ec3440684a6906d28051
```

```json
{
    "action": "edited",
    "projects_v2_item": {
        "id": 8420853,
        "node_id": "PVTI_lADOBay1Kc4AEwi9zgCAffU",
        "project_node_id": "PVT_kwDOBay1Kc4AEwi9",
        "content_node_id": "PR_kwDOGfwnTc48u60R",
        "content_type": "PullRequest",
        "creator": {
            "login": "AlekSi",
            "id": 11512,
            "node_id": "MDQ6VXNlcjExNTEy"
        },
        "created_at": "2022-08-09T20:36:46Z",
        "updated_at": "2022-08-19T17:46:58Z",
        "archived_at": null
    },
    "changes": {
        "field_value": {
            "field_node_id": "PVTSSF_lADOBay1Kc4AEwi9zgCzrKM",
            "field_type": "single_select"
        }
    },
    "organization": {
        "login": "FerretDB",
        "id": 95204649,
        "node_id": "O_kgDOBay1KQ"
    },
    "sender": {
        "login": "AlekSi",
        "id": 11512,
        "node_id": "MDQ6VXNlcjExNTEy"
    }
}
```

The PR in question is #85.

### PR DESCRIPTION
**Pull request by chilagrow**
Closes #88.

### CODE REVIEW

File: org-webhook/main.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+func main() {
+	action := githubactions.New()
+
+	h := newWebhookHandler(action)
+
+	r := mux.NewRouter()
+	r.HandleFunc("/webhook", h.handleWebhook).
+		Methods("POST").
+		Headers("Content-Type", "application/json")
```
Comment (AlekSi) [inline org-webhook/main.go:34]: We could check that with `if`s inside the handler

**Review by AlekSi — COMMENTED** [review]
Looks good overall, but let's drop mux – that server will have a single endpoint, no need to pull an additional dependency

File: org-webhook/main.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+)
+
+// main starts http server. It requires env vars.
+// Example:
+// WEBHOOK_ADDR="localhost:8088" GITHUB_SECRET_KEY={{WebhookSecret}} go run org-webhook/main.go
+func main() {
+	action := githubactions.New()
+
+	h := newWebhookHandler(action)
+	addr := action.Getenv("WEBHOOK_ADDR")
```
[the remark this answers is not in the document]
Comment (chilagrow) [reply org-webhook/main.go]: Fair point, thanks :pray:

File: org-webhook/main.go [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+import (
+	"github.com/google/go-github/v45/github"
+	"github.com/sethvargo/go-githubactions"
+	"log"
+	"net/http"
+)
+
+// main starts http server. It requires env vars.
+// Example:
+// WEBHOOK_ADDR="localhost:8088" GITHUB_SECRET_KEY={{WebhookSecret}} go run org-webhook/main.go
```
[the remark this answers is not in the document]
Comment (chilagrow) [reply org-webhook/main.go]: Added checks :+1:

**Review by AlekSi — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
Looks like a good start!

**Pull request merged**